spec/friendly_shipping/services/rl/parse_rate_quote_response_spec.rb

```diff
@@ -25,7 +25,27 @@
       expect(rate.shipping_method.service_code).to eq("STD")
       expect(rate.warnings).to be_empty
       expect(rate.errors).to be_empty
-      expect(rate.data).to eq({})
+      expect(rate.data).to eq(
+        cost_breakdown: [
+          { "Amount" => "$11.25", "Description" => "Class: 92.5", "Rate" => "$1,124.72", "Weight" => "1" },
+          { "Amount" => "$11.25", "Description" => "Class: 92.5", "Rate" => "$1,124.72", "Weight" => "1" },
+          { "Amount" => "$11.25", "Description" => "Class: 92.5", "Rate" => "$1,124.72", "Weight" => "1" },
+          { "Amount" => "$11.25", "Description" => "Class: 92.5", "Rate" => "$1,124.72", "Weight" => "1" },
+          { "Amount" => "$1,690.72", "Description" => "Minimum Charge", "Type" => "MINIMUM" },
+          { "Amount" => "$1,487.83", "Description" => "R+L Discount Saves This Much", "Rate" => "88%", "Type" => "DISCNT" },
+          { "Amount" => "$202.89", "Description" => "Discounted Freight Charge", "Type" => "DISCNF" },
+          { "Amount" => "$71.62", "Description" => "Fuel Surcharge", "Rate" => "35.3%", "Type" => "FUEL" },
+          { "Amount" => "$110.90", "Description" => "Manhattan Arbitrary Charge", "Type" => "ARBMH" },
+          { "Amount" => "$51.30", "Description" => "Hazmat Fee", "Rate" => "$51.30", "Type" => "HAZM" },
+          { "Amount" => "$436.71", "Description" => "Net Charge", "Type" => "NET" }
+        ]
+      )
+    end
+
+    it "adds non-standard service level charges to cost breakdowns" do
+      non_standard_rates = api_result.data.reject { |rate| rate.shipping_method.service_code == "STD" }
+      non_standard_charges = non_standard_rates.map { |rate| rate.data[:cost_breakdown].last }
+      expect(non_standard_charges).to include({ "Amount" => "$51.30", "Description" => "Guaranteed Service", "Type" => "GSDS" })
     end
```

Comment:
It is weird that R&L returns quotes with the charges in common across service levels. It took me a minute to notice that the `cost_breakdown` is only checked on the first rate. Would it be worth testing that both rates include all of the items in the breakdown?

Reply:
Yes, it's odd for sure. You basically have to take the "Charge" value from each service level above Standard and add it to the "Net Charge" above to arrive at the _actual_ net charge for that service level.

It doesn't matter for our purposes since we're only ever using "Standard" service anyway, but I was thinking about adding the "Charge" for each service level to the "Net Charge" so the total would be correct. I concluded that had the potential to cause confusion and just left things as they are.

Yes, it's probably worth testing the charges for the "Guaranteed" rate as well. I'll update the test.